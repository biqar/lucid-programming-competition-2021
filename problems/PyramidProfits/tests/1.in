3
Yvonne Arthur 20
Jamal Arthur 50
Arthur Jude 30
4
Yvonne 12000
Jamal 8000
Jude 4151
Arthur 500
