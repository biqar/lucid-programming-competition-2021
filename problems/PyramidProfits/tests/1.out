Arthur 4230
Jamal 1200
Jude 18501
Yvonne 720
